vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -820,6 +820,11 @@ EXPORT(int, sceGxmDrawPrecomputed, SceGxmContext *context, SceGxmPrecomputedDraw
     assert(vertex_state);
     assert(fragment_state);
 
+    if (!vertex_state)
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_PRECOMPUTED_VERTEX_STATE);
+    if (!fragment_state)
```

Comment:
If you add check you can so remove assert

Reply:
asserts are there for debugging, there is no need to delete them.

Reply:
i request after add check, look scriban pr, add check = remove assert, no need anymore